build.sbt

```diff
@@ -115,6 +115,23 @@ lazy val decline =
       libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.3.0",
       coverageEnabled := false
     )
+    .nativeSettings(
+      // Note: scala-native does not include a java.nio.time implementation and
+      //       the implementation from https://github.com/ekrich/sjavatime
+      //       is not complete
+      //       (missing 7 definitions while linking -- 16 without sjavatime)
+      libraryDependencies += "org.ekrich" %%% "sjavatime" % "1.1.5",
```

Comment:
IIUC, this is both adding the dependency and excluding the code that will eventually use it... is that right?

(If so, maybe we should just add the new dep to the comment for now?)

Reply:
Yes, that's right. I'll comment this out for now.